contrib/infiniband/infinibandwrappers.c

```diff
@@ -15,7 +15,12 @@
 void *dlopen(const char *filename, int flag) {
   if (filename) {
     if (strstr(filename, "libibverbs.so")) {
-      return RTLD_DEFAULT;
+      void *handle = NEXT_FNC(dlopen)("libdmtcp_infiniband.so", flag);
+      if (handle == NULL) {
+        fprintf(stderr,
+                "Please add $DMTCP_PATH$/lib/dmtcp to LD_LIBRARY_PATH");
```

Comment:
Let's mention both user options here.  I'm also adding extra `\n` and '*'.  This message should stand out, since it's important for the user to see it.
```        fprintf(stderr,
               "\n*** Please either add $DMTCP_PATH$/lib/dmtcp to LD_LIBRARY_PATH,\n"
               "*** or else include an absolute pathname for libdmtcp_infiniband.so\n\n");
```